### DISCUSSION TITLE
Disable SourceUpdate for Monolingual translations

### DISCUSSION THREAD

**Discussion by swurzinger** [WeblateOrg/weblate#5728] [Q&A]
When translating with a monolingual translation files is it possible to disable special features for the base translation:

- changing a base translation string marks this string in all other translations as "needs editing"
- when the base translation string is empty other translations are "readonly"

Especially the first one does not necessarily make sense when dealing with e.g. structured JSON translations with a common key across all languages.

**Comment by nijel** [comment]
When you change the source string, the translations should be changed as well. 

I'm pretty sure there is an issue where this topic has been discussed, but I can't find it right now.
